lib/python/qmk/cli/json2c.py

```diff
@@ -1,6 +1,7 @@
 """Generate a keymap.c from a configurator export.
 """
 import json
+import sys
```

Comment:
Please undo these changes, or submit them in a separate PR.

Reply:
Ofc, I intended these changes for a separate PR. My bad.